docker/packager/binary/build.sh

```diff
@@ -55,6 +55,17 @@ ccache_status
 # clear cache stats
 ccache --zero-stats ||:
 
+# Check whether the directory with pre-build scripts exists and not empty.
+if [ -d "/pre-build" ] && [ -z "$(ls -A /pre-build)" ]; then
+   echo "There are no subcommands to execute :)"
+else
+  # Execute all commands
+  for file in /pre-build/*.sh ;
```

Comment:
Sorry, messed the things

Here it should be 
```suggestion
if [ -d "/build/packages/pre-build" ] && [ -z "$(ls -A /build/packages/pre-build)" ]; then
   echo "There are no subcommands to execute :)"
else
  # Execute all commands
  for file in /build/packages/pre-build/*.sh ;
```

And `docker/packager/pre-build/example.sh` should be `packages/pre-build/example.sh` accordingly